sdk/loadtesting/azure-developer-loadtesting/tests/test_admin_ops.py

```diff
@@ -282,3 +291,105 @@ def test_get_server_metrics_config(self, loadtesting_endpoint, loadtesting_test_
         client = self.create_administration_client(loadtesting_endpoint)
         result = client.get_server_metrics_config(loadtesting_test_id)
         assert result is not None
+
+class TestTestProfileAdministrationClientt(LoadtestingTest):
```

Comment:
In my opinion, for now, lets move this class to a separate file (`test_admin_test_profile_ops.py`) Same for Test Profile Run tests